bee-network/bee-gossip/src/network/host.rs

```diff
@@ -226,12 +226,7 @@ async fn dial_peer(swarm: &mut Swarm<SwarmBehaviour>, peer_id: PeerId, peerlist:
         })
         .expect("peer must exist");
 
-    debug!(
-        "Dialing peer: {} ({}) attempt: #{}.",
-        alias,
-        alias!(peer_id),
-        dial_attempt
-    );
+    debug!("Dialing peer: {} ({}) attempt: #{}.", alias, peer_id, dial_attempt);
```

Comment:
You absolutely don't have to do this, but I think this is easier to parse:
```suggestion
    debug!("Dialing peer: {alias} ({peer_id}) attempt: #{dial_attempt}.");
```

Reply:
Reduced this PR. This will go into the bee-gossip rewrite.